kubectl-rook-ceph.sh

```diff
@@ -344,6 +432,10 @@ function run_main_command() {
 : "${ROOK_CLUSTER_NAMESPACE:=rook-ceph}"
 : "${ROOK_OPERATOR_NAMESPACE:=$ROOK_CLUSTER_NAMESPACE}"
 : "${TOP_LEVEL_COMMAND:=kubectl}"
+: "${RESET:=\033[0m}"                            # For no color
+: "${ERROR_PREFIX:=\033[1;31mError: $RESET}"     # \033[1;31m for Red color
```

Comment:
nit: I think the script will be more easily readable if the space is not here, but is instead added where the prefix is used. (similar change for warning and info)
```suggestion
: "${ERROR_PREFIX:=\033[1;31mError:$RESET}"     # \033[1;31m for Red color
```

Reply:
agreed